spec/components/provider_interface/safeguarding_declaration_component_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+RSpec.describe ProviderInterface::SafeguardingDeclarationComponent do
```

Comment:
We should add another spec to cover when a candidate has entered details for safeguarding e.g. `application_form = build_stubbed(:application_form, safeguarding_issues: 'I have a criminal conviction.')`. This should also display that the candidate has shared information related to safeguarding.

Reply:
👍